refocus-bdk-client.js

```diff
@@ -614,28 +633,30 @@ module.exports = (config) => {
      * @returns {Promise} - Event response
      */
     createEvents: (room, msg, context, type) => {
-      log.debug('Creating a new Event. ', { room, msg, context, type });
-      const events = {
-        log: msg,
-        roomId: room,
-        actionType: type,
-        botId: botName
-      };
-      if (context) {
-        events.context = context;
-        events.context.user = _user;
-      } else {
-        events.context = {
-          user: _user
+      return getBotId(SERVER, TOKEN, botName).then((id) => {
```

Comment:
do you still need this getbotid or could you pass in the botname?

Reply:
Need the botId I think, the realtime app uses the botIds for socket namespaces